libtest/libtest_allmsgs/rt-test.go

```diff
@@ -22,7 +22,7 @@ func RTTest(t *testing.T) {
 		t.Error("Failed to make node;", err)
 		return
 	}
-	node.Logger().SetSeverity(ros.LogLevelWarn)
+	node.SetLogLevel(1)
```

Comment:
What is this nonsense!?

Reply:
Forgot to update in yesterdays PR - want to keep same log level?

Reply:
Can you just fix this last one (it's in `libtest/libtest_allmsgs/rt-test.go`), then looks good.